### PR TITLE
Enable conversion calculations

### PR DESCRIPTION
**Pull request by Eleonore9**
* Fix function names inversion in pipeline.clj
* (Replaced edn/read-string by Double/valueOf)
* Fix args inverted in conversion-fn for both conversions
* Select sensors to be converted to co2
* Check sensor-id for sensors to be converted
* (Formatting)

### CODE REVIEW

**Comment by acron0** [comment]
:shipit:

File: src/clj/kixi/hecuba/controller/pipeline.clj [hunk 1]
```diff
@@ -157,10 +157,14 @@
...
-            {:keys [unit device_id type]} sensor]
+            {:keys [unit device_id type period]} sensor
+            substring? (fn [sub st] (not= (.indexOf st sub) -1))
+            regex-seq  ["oilConsumption" "gasConsumption"]
+            should-convert-type? (fn [type] (some #(substring? % type) regex-seq))]
         (when (and unit
                    (some #(= (.toUpperCase unit) (.toUpperCase %)) ["m^3" "ft^3"])
-                   (some #(= type %) ["gasConsumption" "oilConsumption"]))
+                   (= period "PULSE")
+                   (should-convert-type? type))
```
Comment (otfrom) [inline src/clj/kixi/hecuba/controller/pipeline.clj]: I really like how you've done this function.

**Comment by otfrom** [comment]
:shipit:

File: src/clj/kixi/hecuba/controller/pipeline.clj (hunk 1 above)
Comment (Eleonore9) [inline src/clj/kixi/hecuba/controller/pipeline.clj]: Thanks! I got inspired by a neighbouring function

**Pull request merged**